core/webviz/_math.py

```diff
@@ -0,0 +1,58 @@
+import jinja2
+import markdown
+from ._page_element import PageElement
+from ._header_element import HeaderElement
+
+
+class MathJaxPattern(PageElement):
+    def __init__(self, md):
+        self._md = md
+        super(MathJaxPattern, self).__init__()
+
+        extension = markdown.Markdown(
+            extensions=['mdx_math'],
+            extension_configs={
+                'mdx-math': {'enable_dollar_delimiter': True}
+            }
+        )
+
+        self.header_elements.add(HeaderElement(
+            tag='script',
+            attributes={
+                'type': 'text/x-mathjax-config'
+            },
+            content="""
+                MathJax.Hub.Config({
+                    jax: ["input/TeX","input/MathML","output/CommonHTML"],
+                    extensions: [
+                        "tex2jax.js",
+                        "mml2jax.js",
+                        "MathMenu.js",
+                        "MathZoom.js",
+                        "AssistiveMML.js",
+                         "a11y/accessibility-menu.js"
+                    ],
+                    TeX: {
+                        extensions: [
+                            "AMSmath.js",
+                            "AMSsymbols.js",
+                            "noErrors.js",
+                            "noUndefined.js"
+                        ]
+                    }
```

Comment:
If the `TeX` dictionary is changed to this we get numbering of equations (which then can be referenced):

```
                    TeX: {
                        equationNumbers: {autoNumber: "all"},
                        extensions: [
                            "AMSmath.js",
                            "AMSsymbols.js",
                            "noErrors.js",
                            "noUndefined.js"
                        ]
                    }
```